### PR TITLE
Datapool fgen clean

### PR DESCRIPTION
**Pull request by jeticg**
All languages except Arabic Finnish and Persian, the data for these
languages are missing or wrong

### CODE REVIEW

**Comment by anoopsarkar** [comment]
Have we confirmed that the performance of the 1st order features is identical to the performance of master on the penn-wsj-deps dataset. is there a log file in this branch to prove this.

**Comment by jeticg** [comment]
I am not familiar with the methods of performance measurement in this case, I think it must have something to do with the datasets. The 10 languages each took different time to finish the test, the times are all recorded in the log files.

**Comment by anoopsarkar** [comment]
I am not too worried about the time at the moment (unless it is double or something than the previous version in which case I do care). I am more concerned about the UAS (unlabeled attachment accuracy) on the test set.

**Comment by jeticg** [comment]
Seanla instructed me to test all the languages except for English, so I don't really have the times of the tests on previous versions. But the test on German dataset did took more than four hours, the average of the rest are about 1-2 hours. I sure hope that's the result we are expecting. The average UAS is about 0.85-0.9, with the lowest ones be 0.75 and 0.79, highest ones be 9.4

**Comment by anoopsarkar** [comment]
All those tests are great and useful. But before we merge this branch I would like to double check that the performance on penn-wsj-deps which is in `/cs/natlang-projects/glm-parser/penn-wsj-deps` is identical to the performance of the current master branch.

Train sections: `2-21`
Test on Dev which is sections `0,1,22,24` and Test which is section `23`

**Comment by jeticg** [comment]
OK, I'll inform Sean and perform the tests tonight after finishing my homework. I'll submit the results by tomorrow morning.

**Comment by anoopsarkar** [comment]
Do it when you are able to find time. Let us merge after we know the results.

**Comment by anoopsarkar** [comment]
If you compare the performance in https://github.com/sfu-natlang/glm-parser/blob/master/logs/2014-06-08/avg_perceptron_results.txt

    Training data: section 02-21
    Testing data: section 0,1,22,24
    Highest dependency accuracy: 0.908843846900 (151826, 167054)
    Unlabeled Attachment Accuracy: 0.91248979967 (158786, 174014)

The current performance in https://github.com/sfu-natlang/glm-parser/tree/datapool_fgen_clean/logs/2016-03-09 is not nearly as good. Perhaps the test was not identical, in which case we should repeat it to be identical to previously best known result in the master branch.

**Comment by Travelinglight** [comment]
The current performance in [https://github.com/sfu-natlang/glm-parser/tree/datapool_fgen_clean/logs/2016-03-09](https://github.com/sfu-natlang/glm-parser/tree/datapool_fgen_clean/logs/2016-03-09) is based on (training section 02, testing section 00), so the accuracy is only about 0.8459, which is the same as the master version. Sean would test on the whole dataset today, hope it'll have this 0.912 accuracy.
